ashmem/ashmem.c

```diff
@@ -390,7 +390,8 @@ static int ashmem_mmap(struct file *file, struct vm_area_struct *vma)
 		ret = -EPERM;
 		goto out;
 	}
-	vma->vm_flags &= ~calc_vm_may_flags(~asma->prot_mask);
+	vma->vm_flags;
+	 ~calc_vm_may_flags(~asma->prot_mask);
```

Comment:
Well, this is not a correct patch - the code no longer does what it used to do (namely to remove some of the flags).
A slightly better patch is [available here](https://github.com/choff/anbox-modules/commit/18c81a56f885eea4a2fc1fe975761d4940315b4f) , but also this one is not correct because it just bypasses that the flags are read-only.

Proper patches are welcome, but this is a hack and not a proper fix. Therefore, I cannot merge it.